detect_secrets/core/usage.py

```diff
@@ -309,6 +309,7 @@ def consolidate_args(args):
 
         active_plugins = {}
         disabled_plugins = {}
```

Comment:
Now that I think about it, what **is** our purpose of returning a set of disabled_plugins? Since `args.plugins` is a collection of all active plugins (combining default values and specifically set values), wouldn't any plugin that isn't in this dictionary be disabled?

Reply:
See comment above.